docs/rules/constructor-super.md

```diff
@@ -10,7 +10,9 @@ This rule checks whether or not there is a valid `super()` call.
 
 This rule is aimed to flag invalid/missing `super()` calls.
 
-The following patterns are considered problems:
+## Examples
```

Comment:
Whether for good or for bad, so far the examples of code for default options (especially for rules with no other options) have been directly under h2 Rule Details. Can you delete h2 Examples for now and add a comment to the guidelines issue with your thoughts about this (unless you have already ;).